atomicapp/providers/docker.py

```diff
@@ -30,6 +32,13 @@ class DockerProvider(Provider):
     key = "docker"
 
     def init(self):
+        self.namespace = DEFAULT_NAMESPACE
+        self.default_name = DEFAULT_CONTAINER_NAME
+
+        logger.debug("Given config: %s", self.config)
+        if self.config.get("namespace"):
+            self.namespace = self.config.get("namespace")
+        logger.debug("DEFAULT NAMESPACE: %s", self.namespace)
```

Comment:
I don't think this prints "Default namespace" as it can be replaced by user value